### ISSUE TITLE
Reproduce results

### ISSUE THREAD

**Issue by luochenyan-rgb** [winycg/CIRKD#4]
Hello, in the process of reproducing the program, deeplabv3-resnet101 achieved similar results to the original paper, but when deeplabv3-resnet18 is used as the baseline and kd, the miou value differs by 3%-4%, whether the small model has different training Strategy?

**Comment by winycg** [comment]
Thanks for your attention. In our implementation, the teacher network is trained by 80K iterations with a batch size of 8, the student network is trained by 40K iterations with a batch size of 16. Do you follow the same setup?

**Comment by luochenyan-rgb** [comment]
Hi, thanks for your reply, I still have the following questions to ask you.
Is the batch size setting based on 8 GPUs?
If using 3 GPUs and the batch size of each GPU corresponds to the data you provided, do I need to modify the value of the initial learning rate? Are the initial learning rates of the baseline and KD network 0.2?

**Comment by winycg** [comment]
Hi, we conduct experiments on 8 GPUs with a batch size of 16, i.e. 2 samples per GPU.
The initial learning rate can be linearly scaled with the batch size, e.g. 0.02 with 16 or 0.01 with 8. If you use 3 GPUs, the default code will split the batch size of 16 into 3 sub batches, i.e. 5~6  samples per GPU.

**Comment by winycg** [comment]
Hi, we conduct experiments on 8 GPUs with a batch size of 16, i.e. 2 samples per GPU.
The initial learning rate can be linearly scaled with the batch size, e.g. 0.02 with batch size 16 or 0.01 with batch size 8. If you reduce the batch size, you should increase the training iterations with the same times, e.g. 40K iters for batch size 16 or 80K iters for batch size 8.
If you use 3 GPUs, the default code will split the batch size of 16 into 3 sub batches, i.e. 5~6 samples per GPU. If the batch size is still 16, I think it do not need to modify other settings.

**Comment by luochenyan-rgb** [comment]
Thank you very much for your guidance, the current reproduced results have achieved similar results to the original paper.
